Firestore/core/src/firebase/firestore/timestamp.cc

```diff
@@ -84,8 +85,13 @@ Timestamp Timestamp::FromTimePoint(
 #endif  // !defined(_STLPORT_VERSION)
 
 std::string Timestamp::ToString() const {
-  return std::string("Timestamp(seconds=") + std::to_string(seconds_) +
-         ", nanoseconds=" + std::to_string(nanoseconds_) + ")";
+  return absl::StrCat("Timestamp(seconds=", seconds_,
+                      ", nanoseconds=", nanoseconds_, ")");
+}
+
+std::ostream& operator<<(std::ostream& out, const Timestamp& timestamp) {
+  out << timestamp.ToString();
+  return out;
```

Comment:
`return out << timestamp.ToString();` ?

Reply:
Done. I keep forgetting that I can do that.